packages/ui/src/components/Channel/index.tsx

```diff
@@ -285,28 +285,27 @@ export function Channel({
                           <NegotionMismatchNotice />
                         )}
                         {!isChatChannel && canWrite && !showGalleryInput && (
-                          <View position="absolute" bottom="$l" right="$l">
-                            {uploadInfo.uploadedImage &&
-                            uploadInfo.uploading ? (
-                              <View alignItems="center" padding="$m">
-                                <Spinner />
-                              </View>
-                            ) : (
-                              <FloatingActionButton
-                                onPress={() =>
-                                  uploadInfo.uploadedImage
-                                    ? messageSender([], channel.id)
-                                    : setShowAddGalleryPost(true)
-                                }
-                                icon={
-                                  uploadInfo.uploadedImage ? (
-                                    <ArrowUp />
-                                  ) : (
-                                    <Add />
-                                  )
-                                }
-                              />
-                            )}
+                          <View
+                            position="absolute"
+                            bottom={bottom}
+                            flex={1}
+                            width="100%"
+                            alignItems="center"
```

Comment:
I think this is fine, but would you mind verifying that the container won't block touches on elements underneath?

Reply:
Can do, I'll follow-up if so